cms/static/js/index.js

```diff
@@ -159,6 +159,7 @@ define(['domReady', 'jquery', 'underscore', 'js/utils/cancel_on_escape', 'js/vie
             $cancelButton.bind('click', makeCancelHandler('library'));
             CancelOnEscape($cancelButton);
 
+            CreateLibraryUtils.setupOrgAutocomplete();
```

Comment:
Why setup? Why not only `orgAutoComplete()`
```suggestion
            CreateLibraryUtils.orgAutoComplete();
```

Reply:
Just following the same convention that is being used in this file 
https://github.com/colaraz/edx-platform/blob/cb4213cfb14a406f18934bc8c4fc67c41a0968a6/cms/static/js/index.js#L123